rust/src/dns/log.rs

```diff
@@ -22,9 +22,6 @@ use std::string::String;
 use crate::dns::dns::*;
 use crate::jsonbuilder::{JsonBuilder, JsonError};
 
-pub const LOG_QUERIES: u64 = BIT_U64!(0);
```

Comment:
@jasonish should these get used ?

Reply:
It'll be used in the final push to remove DNS from the C side.

Reply:
Should it be removed now, so that the commit "to remove DNS from the C side" is a logical unit ?